CHANGELOG.md

```diff
@@ -2,10 +2,34 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.6 August 11, 2021
+# 10.0.0-rc.7 August 25, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Breaking changes ⚠️
+* Remove the expression getter/setters for source properties. ([#568](https://github.com/mapbox/mapbox-maps-android/pull/568))
+
+## Features ✨ and improvements 🏁
+* Add generateId property for GeoJsonSource. ([#538](https://github.com/mapbox/mapbox-maps-android/pull/538))
+* Add default value to improve usability of FeatureState API. ([#588](https://github.com/mapbox/mapbox-maps-android/pull/588))
+* Add Style#moveStyleLayer(layerId: String, layerPosition: LayerPosition?): Expected<String, None> API ([#563](https://github.com/mapbox/mapbox-maps-android/pull/563))
+* Allow using combination of line-dasharray and line-gradient for line layer.
+
+## Bug fixes 🐞
+* Fix snapshotter leak. Now calling MapSnapshotter#destroy truly frees memory. ([#569](https://github.com/mapbox/mapbox-maps-android/pull/569))
```

Comment:
I think this PR is overwritten by #571 below, right? cc @kiryldz

Reply:
Yep, guess you're right and this line can be removed.
Thanks for catching!